tests/helpers/test_template.py

```diff
@@ -801,7 +795,7 @@ def test_is_state(hass):
         """,
         hass,
     )
-    assert tpl.async_render() == "False"
+    assert tpl.async_render() is False
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)